goreq.go

```diff
@@ -270,7 +271,18 @@ func (r *Request) AddCookie(c *http.Cookie) {
 func (r Request) WithCookie(c *http.Cookie) Request {
 	r.AddCookie(c)
 	return r
+}
 
+func (r *Request) AddProxyHeader(name string, value string) {
```

Comment:
Maybe change this to `AddProxyConnect` header? Because these headers will be only sent during the `connect` phase according to the golang doc. Same for `WithProxyHeader`

Reply:
ok.
Change the method name.